src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,362 @@
+
+The following objects are required for the sidecar to successfully report data:
+
+* The `ClusterRole` providing the permission needed by the `nri-kubernetes integration`
+* A `ClusterRoleBinding` linking the `ClusterRole` and the service account of the pod
+* The secret storing the New Relic `licenseKey` in each Fargate namespace 
+* The sidecar container in the spec template of the monitored workload
+
+#### Manual injection installation [#manual-injection-install]
+
+<Callout variant="tip">
+These manual setup steps are for a generic installation. Before completing these, take a look at the [Configuration](#config-manual) section below to see if you want to modify any aspects of the automatic injection.
+</Callout>
+
+Complete the following for manual injection:
+
+1. If `ClusterRole` does not exist, create it and grant the permissions required to hit the metrics endpoints. This only needs to be done once, even for monitoring multiple applications in the same cluster.
+
+    <CollapserGroup>
+      <Collapser
+        id="cluster-role"
+        title="clusterRole.yaml"
+      >
+      You can use this snippet as it appears below, without any changes:
+
+      ```      
+      apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRole
+      metadata:
+        labels:
+          app: newrelic-infrastructure
+        name: newrelic-newrelic-infrastructure-infra-agent
+      rules:
+      - apiGroups:
+        - ""
+        resources:
+        - nodes
+        - nodes/metrics
+        - nodes/stats
+        - nodes/proxy
+        - pods
+        - services
+        verbs:
+        - get
+        - list
+      - nonResourceURLs:
+        - /metrics
+        verbs:
+        - get
+      ```
+      </Collapser>
+    </CollapserGroup>
+
+2. For each workload you want to monitor, add an additional sidecar container for the `newrelic/infrastructure-k8s` image. Here is an example of an injected sidecar. 
+
+    <CollapserGroup>
+      <Collapser
+        id="container-to-inject"
+        title="containerToInject.yaml"
+      >
+      [PLEASE INSERT INSTRUCTIONS ABOUT HOW TO USE THIS SNIPPET]
+
+        ```
+        apiVersion: apps/v1
+        kind: Deployment
+        spec:
+          template:
+            spec:
+              containers:
+              - name: newrelic-infrastructure
+                env:
+                - name: NRIA_LICENSE_KEY
+                  valueFrom:
+                    secretKeyRef:
+                      key: license
+                      name: newrelic-newrelic-infrastructure-config
+                - name: NRIA_VERBOSE
```

Comment:
Add as well to the envVars list: 
```
- name: DISABLE_KUBE_STATE_METRICS
   value: "true"
```
(the order is not important)

Reply:
@paologallinaharbur, I inserted it here:

```
                - name: NRIA_VERBOSE
                    value: "1"
                - name: DISABLE_KUBE_STATE_METRICS
                    value: "true"
                - name: CLUSTER_NAME
                    value: testing-injection
```

Reply:
note that the indentions wants always `value` under `name`, but I guess it is the github preview messing it up!

Reply:
@paologallinaharbur, `value` is indented the same as `name` in the snippet, so I think we're OK.